nemo/utils/data_utils.py

```diff
@@ -127,22 +131,39 @@ def ais_endpoint_to_dir(endpoint: str) -> str:
 
 
 def ais_binary() -> str:
-    """Return location of `ais` binary."""
+    """Return location of `ais` binary if available."""
     path = shutil.which('ais')
 
     if path is not None:
         logging.debug('Found AIS binary at %s', path)
         return path
 
-    logging.warning('AIS binary not found with `which ais`.')
-
     # Double-check if it exists at the default path
     default_path = '/usr/local/bin/ais'
     if os.path.isfile(default_path):
         logging.info('ais available at the default path: %s', default_path)
         return default_path
     else:
-        raise RuntimeError('AIS binary not found.')
+        logging.warning(
+            f'AIS binary not found with `which ais` and at the default path {default_path}.', mode=LogMode.ONCE
+        )
+        return None
+
+
+@lru_cache(maxsize=1)
```

Comment:
@monica-sekoyan let's not copy the code from lhotse into nemo for this purpose. This would cause a lot of headaches later as we'll have to modify AIS related functionality in both repos. 

It'd be better to instead move/copy the necessary functions from `nemo/utils/data_utils.py` into `nemo/collections/asr/...` collection and import lhotse there, avoiding lhotse import outside of ASR collection.